nucliadb/nucliadb/search/requesters/utils.py

```diff
@@ -186,6 +186,7 @@ async def node_query(
             errors.capture_exception(result)
             await abort_transaction()
             logger.exception("Error while querying shard data", exc_info=True)
+            logger.info(result)
```

Comment:
Why do we need this log?

Reply:
removed comment and updated branch so we can merge and close this issue out.